code/modules/antagonists/cult/cult.dm

```diff
@@ -431,6 +433,6 @@
 			return FALSE
 	else
 		return FALSE
-	if(HAS_TRAIT(M, TRAIT_MINDSHIELD) || issilicon(M) || isbot(M) || isdrone(M) || !M.client)
```

Comment:
This one has been definitively identified by multiple people as causing unintended and frustrating situations.
I'd even go so far as to say other antagonist types should be off limits as well, as we also had a heretic cultist.